RELEASES.md

```diff
@@ -6,3 +6,9 @@
 ### 3.0.1
 - Added support for caching from GHES 3.5.
 - Fixed download issue for files > 2GB during restore.
+
+### 3.0.2
+- Added support for dynamic cache size cap on GHES.
+
+### 3.0.3
```

Comment:
why are we doing 2 releases?

Reply:
We're doing only one release i.e. 3.0.3. We had missed updating release notes for the previous release, hence added that information here just to track the same.

Reply:
Ah nice. you might also want to add these to README.md. I'll suggest removing v2 from there now as they are quite old.

Reply:
Alright, let me do that as well. Will also update the link for 

> Refer [here](https://github.com/actions/cache/blob/v1/README.md) for previous versions 


to [v2](https://github.com/actions/cache/blob/v2/README.md) as it is currently pointing to [v1 README](https://github.com/actions/cache/blob/v1/README.md).